src/modules/map/components/olMap/handler/measure/OlMeasurementHandler.js

```diff
@@ -380,8 +406,9 @@ export class OlMeasurementHandler extends OlLayerHandler {
 	async _save() {		
 		const options = { featureProjection: 'EPSG:3857', rightHanded: true, decimals: 8 };
 		const format = new KML({ writeStyles: true });
+		const features = this._vectorLayer.getSource().getFeatures();		
+		features.forEach(f => this._overlayManager.saveManualOverlayPosition(f));
 		this._storedContent = format.writeFeatures(this._vectorLayer.getSource().getFeatures(), options);
```

Comment:
I think it would be useful if we refactor all import/export KML logic into a separate file e.g.`/formats/kml.js`.
In the future, we can add here additional logic, like setting a documents name when exporting
```
 kmlString = kmlString.replace(/<Document>/, `<Document><name>${name}</name>`);
```
Later also possible format files are gpx and geoJson...